http/src/request/channel/stage/create_stage_instance.rs

```diff
@@ -102,12 +102,13 @@ impl<'a> CreateStageInstance<'a> {
     }
 
     fn start(&mut self) -> Result<()> {
-        self.fut.replace(Box::pin(self.http.verify(Request::from(
-            Route::CreateStageInstance {
-                channel_id: self.channel_id.0,
-                topic: self.topic.clone(),
-            },
-        ))));
+        self.fut
+            .replace(Box::pin(self.http.verify(Request::from_route(
+                Route::CreateStageInstance {
+                    channel_id: self.channel_id.0,
+                    topic: self.topic.clone(),
+                },
+            ))));
```

Comment:
Was the wrong conflict resolved here or did I forget to update these to the new builder methods

Reply:
https://github.com/twilight-rs/twilight/commit/a06c6db89b3ac25cceb2c2f30b46eaf342f5d6b1#diff-bcbe87877441827eaa1ff961fd20c765cc2586f7cee79d01a10e889ed2db2388R105